paper.md

```diff
@@ -66,14 +66,13 @@ To support and disseminate the use of our integration work, we also planned to c
 
 # DPIA in DSW
 
-*TODO: (Pinar) and Vilém - motivation for having DPIA KM+template in DSW, core idea and advantages*
+Writing a data protection impact assessment for a scientific project is a challenging task. Input from multiple key groups is required to obtain complete and precise information. In addition to data stewards, which are commonly the ones in charge of the information collection since their knowledge overlaps all other involved domains, the process involves researchers themselves bringing knowledge of about the actual data and related scientific processes, data managers and IT staff describing the local IT infrastructures and external collaborators reporting on the shared data processing tasks. Finally, legal representatives (e.g. data protection officer) validates the resulting document with existing contracts, institutional policies and approves the final version.
```

Comment:
In addition to data stewards, which are commonly in charge of the information collection since their expertise overlaps all other concerned domains, the process  involves researchers themselves bringing their own knowledge about the data involved and the related scientific processes. Data managers and IT staff who are knowledgeable about the local IT infrastructure, and external collaborators may also report on the shared data processing tasks.

Reply:
Finally, legal representatives (e.g. data protection officer**s**) validate the resulting document using existing contracts and institutional policies.